src/animation/KeyframeTrack.js

```diff
@@ -467,6 +453,16 @@ Object.assign( KeyframeTrack.prototype, {
 
 	}
 
+}
+
+Object.assign( KeyframeTrack.prototype, {
+
+	TimeBufferType: Float32Array,
```

Comment:
I debated whether or not to preserve these prototype properties. They should reduce memory footprint with a large number of animation tracks, though I don't know if that matters in real world usage. They seem better suited as static props than instance props? Anyways, let me know what direction is good and I can tweak this PR as needed.

Honestly, all of the inheritance doesn't seem that useful. In the long run maybe the track system could just be refactored with a more data-driven approach.